src/commands/edge_app_server.rs

```diff
@@ -34,16 +45,16 @@ pub async fn run_server(
             },
         );
 
-    let secrets_map: HashMap<_, _> = secrets.into_iter().collect();
+    let secrets_map: BTreeMap<_, _> = secrets_val.into_iter().collect();
```

Comment:
Hmm. Maybe better to just use Vec<>? It preserves the order and we don't access the values by key/value

Reply:
I think that could be left as it is.

Reply:
We are not using it as a map but OK

Reply:
I'm with Sergey on this one, even though BTreeMap might work, if we can also just use Vec, that would be preferrable.

Reply:
Yeah, I know but there is a great possibility we could use it soon. Also, it doesn't create any dev overhead. That's why I opted to leave as it is.

Reply:
@MuratUrsavas what possibility? I am not aware of it

Reply:
BTree is usually used in the databases(indexes) for fast reads(searches) specifically, there is usually no reason using it in the day to day code, unless there are a lot of items in the memory you want to search in fast. So I don't see any pros using it here.

Reply:
OK, guys, you won. I've converted most of the BTreeMaps to Vec's but kept in YAML conversion. Without a custom deserializer, I cannot change it to a Vec. It will be done in another task.